.github/workflows/merge.yml

```diff
@@ -240,6 +241,15 @@ jobs:
       #    BlackDuck_Api_Token: "${{ secrets.ORION_BLACKDUCK_TOKEN }}"
       #    BlackDuck_Url: "${{ inputs.blackduck_server_url }}"
```

Comment:
If adding the citadel step to this, then we can remove the blackduck one.

Reply:
we just kept Blackduck commented just for future use